qiskit_nature/second_q/mappers/parity_mapper.py

```diff
@@ -14,27 +14,52 @@
 
 from __future__ import annotations
 
+import logging
+
 from functools import lru_cache
 from typing import Union, List
 
 import numpy as np
 
 from qiskit.opflow import PauliSumOp
-from qiskit.quantum_info.operators import Pauli
+from qiskit.quantum_info.analysis.z2_symmetries import Z2Symmetries
+from qiskit.quantum_info.operators import Pauli, PauliList, SparsePauliOp
 
 from qiskit_nature.second_q.operators import FermionicOp
 from .fermionic_mapper import FermionicMapper
 
+logger = logging.getLogger(__name__)
+
 
 class ParityMapper(FermionicMapper):  # pylint: disable=missing-class-docstring
-    def __init__(self):
+    def __init__(self, num_particles: tuple[int, int] | None = None):
         """The Parity fermion-to-qubit mapping.
 
-        When using this mapper ``two_qubit_reduction`` can optionally be used for the qubit
-        operator that is created, see converter class
-        :class:`~qiskit_nature.second_q.mappers.QubitConverter`.
+        When using this mapper, :attr:`num_particles` can optionally be used to apply an additional step
+        of reduction after the mapping to pauli operators. The two-qubit reduction tapers two qubits
+        (middle and last qubit) when the spin orbitals are ordered in two spin sectors (block spin order)
+        and selects the symmetry sector corresponding to the provided number of particles.
```

Comment:
It's better now, but how about this:
```suggestion
        When using this mapper, :attr:`num_particles` can optionally be used to apply an additional step
        of reduction after the mapping to pauli operators. The two-qubit reduction tapers two qubits
        (middle and last qubit) because the spin orbitals are ordered in two spin sectors (block spin order).
        Based on the provided number of particles this allows the automatic selection of
        the correct symmetry sector.
```

The reason I suggest this is because the ordering cannot be configured in Qiskit Nature.